common/lib/xmodule/xmodule/js/spec/video/video_accessible_menu_spec.js

```diff
@@ -112,62 +109,47 @@
 
                 beforeEach(function () {
                     state = jasmine.initializePlayer();
-                    videoTracks = $('li.video-tracks');
-                    container = videoTracks.children('div.a11y-menu-container');
-                    button = container.children('a.a11y-menu-button');
-                    menuList = container.children('ol.a11y-menu-list');
-                    menuItems = menuList.children('li.a11y-menu-item');
-                    menuItemsLinks = menuItems.children('a.a11y-menu-item-link');
+                    videoTracks = $('.video-tracks');
+                    container = videoTracks.children('.wrapper-more-actions');
+                    button = container.children('.button-more.has-dropdown');
+                    menuList = container.children('.dropdown-menu');
+                    menuItems = menuList.children('.dropdown-item');
+                    menuItemsLinks = menuItems.children('.action');
                     spyOn($.fn, 'focus').andCallThrough();
                 });
 
-                it('open/close the menu on mouseenter/mouseleave', function () {
-                    container.mouseenter();
-                    expect(container).toHaveClass('open');
-                    container.mouseleave();
-                    expect(container).not.toHaveClass('open');
-                });
-
-                it('do not close the menu on mouseleave if a menu item has ' +
-                    'focus', function () {
-                    // Open menu. Focus is on last menu item.
-                    container.trigger(keyPressEvent(KEY.ENTER));
-                    container.mouseenter().mouseleave();
-                    expect(container).toHaveClass('open');
-                });
-
-                it('close the menu on click', function () {
-                    container.mouseenter().click();
-                    expect(container).not.toHaveClass('open');
+                it('open the menu on click', function () {
```

Comment:
Nit: these test descriptions should read like natural language. So it should be it "opens the menu on click" instead of it "open the menu on click".